app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,61 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    redirect '/login' unless session[:current_user]
+    @tweets = Tweet.all
+    erb :'/tweets/index' 
+  end
+
+  get '/tweets/new' do
+    if session[:current_user]
+      erb :'tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    redirect '/tweets/new' if params[:content].empty?
+    tweet = Tweet.new
+    tweet.content = params[:content]
+    session[:current_user].tweets << tweet
+    tweet.save
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```